build/common/openwisp/celery.py

```diff
@@ -3,9 +3,28 @@
 import os
 
 from celery import Celery
+from celery.schedules import crontab
 
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'openwisp.settings')
 
 app = Celery('openwisp')
 app.config_from_object('django.conf:settings', namespace='CELERY')
 app.autodiscover_tasks()
+
+app.conf.beat_schedule = {
+    'radius-periodic-tasks': {
```

Comment:
what happens if the radius module is not being used?

Reply:
Added a variable `SET_RADIUS_TASKS` which when set to `False` (i.e when radius container isn't used) will not add radius celery tasks.